### PR TITLE
fix Apple's Terminal naming bug

### PR DESCRIPTION
**Pull request by hijack111**
A quick fix on #46

### CODE REVIEW

File: ttygif.c [hunk 1]
```diff
@@ -121,7 +121,7 @@ take_snapshot_darwin(const char *img_path, Options o)
     static char cmd [256];
 
     if (sprintf(cmd,
-            "screencapture -l$(osascript -e 'tell app \"%s\" to id of window 1') -o -m %s &> /dev/null",
+            "screencapture -l$(osascript -e 'tell app \"%s\" to id of window 1' 2> /dev/null) -o -m %s &> /dev/null",
```
Comment (icholy) [inline ttygif.c]: Why pipe to dev null?

File: ttygif.c [hunk 2]
```diff
@@ -279,13 +279,19 @@ main (int argc, char **argv)
...
     options.skip_limit = 5;
     options.skip_threshold = 0;
     options.window_id = getenv("WINDOWID");
-    options.terminal_app = getenv("TERM_PROGRAM");
+    term_program = getenv("TERM_PROGRAM");
+    if (strcmp(term_program, "Apple_Terminal") == 0) {
+        options.terminal_app = "Terminal.app";
+    } else {
+        options.terminal_app = term_program;
+    }
```
Comment (icholy) [inline ttygif.c]: I'd prefer this in the `#ifdef`

``` c
    if (strcmp(options.terminal_app, "Apple_Terminal") == 0) {
        options.terminal_app = "Terminal.app";
    }
```

File: ttygif.c (hunk 1 above)
Comment (hijack111) [inline ttygif.c]: My OS X always prints a such line into stderr

    2016-03-29 10:12:12.637 osascript[5895:191750] ApplePersistence=NO

I have no idea how to turn it off

File: ttygif.c [hunk 3]
```diff
@@ -279,13 +279,20 @@ main (int argc, char **argv)
...
     options.skip_threshold = 0;
     options.window_id = getenv("WINDOWID");
-    options.terminal_app = getenv("TERM_PROGRAM");
+    term_program = getenv("TERM_PROGRAM");
+    options.terminal_app = term_program;
+#ifdef OS_DARWIN
+    if (strcmp(term_program, "Apple_Terminal") == 0) {
+        options.terminal_app = "Terminal.app";
+    }
+#endif
```
Comment (icholy) [inline ttygif.c]: There's no need for the `term_program` variable.

**Pull request merged**